duo-universal-sdk/src/main/java/com/duosecurity/Utils.java

```diff
@@ -33,40 +33,32 @@
   static String createJwt(String clientId, String clientSecret, String aud) throws DuoException {
     Date expiration = new Date();
     expiration.setTime(expiration.getTime() + ONE_HOUR_IN_MILLISECONDS);
-    try {
-      return JWT.create()
-                  .withHeader(HEADERS)
-                  .withIssuer(clientId)
-                  .withSubject(clientId)
-                  .withAudience(aud)
-                  .withExpiresAt(expiration)
-                  .withJWTId(generateJwtId(32))
-                  .sign(Algorithm.HMAC512(clientSecret));
-    } catch (UnsupportedEncodingException e) {
-      throw new DuoException(e.getMessage(), e);
-    }
+    return JWT.create()
+            .withHeader(HEADERS)
+            .withIssuer(clientId)
+            .withSubject(clientId)
+            .withAudience(aud)
+            .withExpiresAt(expiration)
+            .withJWTId(generateJwtId(32))
+            .sign(Algorithm.HMAC512(clientSecret));
   }
 
   static String createJwtForAuthUrl(String clientId, String clientSecret, String redirectUri,
```

Comment:
Ditto the DuoException question.  I feel like this should have been caught by the build?